tools/lib.sh

```diff
@@ -75,3 +75,7 @@ function contains_element() {
   return 1
 }
 
+function validate_config() {
+  log "Validating json config file $1"
+  python -m json.tool $1 1>/dev/null 
```

Comment:
Yeah, in the interest of getting this merged, let's nick this and add the functionality into osqueryi or something. then, we can add this back to osqueryctl.